src/acesrender.cpp

```diff
@@ -144,7 +144,7 @@ int AcesRender::readCameraSenPath( const char * cameraSenPath,
 
             for ( vector<string>::iterator file = cFiles.begin( ); file != cFiles.end( ); ++file ) {
                 string fn( *file );
-                
+                if ( fn.find(".json") == std::string::npos ) continue;
```

Comment:
The common method of "opendir (path)" will open a path and return a list of files in that path to "cFiles". I may be able to find a method that only returns files of a certain type such as JSON.